tests/test_pipenv.py

```diff
@@ -271,6 +271,28 @@ def test_uninstall_all_local_files(self):
             assert 'tablib' in c.out
             assert 'tablib' not in p.pipfile['packages']
 
+    @pytest.mark.run
+    @pytest.mark.uninstall
+    def test_uninstall_all_dev(self):
+        with PipenvInstance() as p:
+            c = p.pipenv('install --dev requests pytest')
```

Comment:
Not sure if you're still working on this and if I'm just prematurely reviewing but you may want to install some non-dev stuff and test that it isn't getting removed/deleted

Reply:
Good thought